resources/template/menu.php

```diff
@@ -0,0 +1,112 @@
+<?php
+namespace template;
+require $_SERVER['DOCUMENT_ROOT'] . '/../vendor/autoload.php';
+use src\ProjectConfig;
+class Menu{
+    private $role;
+    private $uname;
+    private $current_file;
+    public $paths;
+
+    /*
+        * $role could have values: 'anon','user','admin'
```

Comment:
Create an interface to map the roles with constants, and include that with user class, This can done in a followup.

Reply:
Yes I'll open an issue for this.